Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollbarsController.h

```diff
@@ -56,6 +56,10 @@ class RemoteScrollbarsController final : public WebCore::ScrollbarsController {
     int minimumThumbLength(WebCore::ScrollbarOrientation) final;
     void updateScrollbarEnabledState(WebCore::Scrollbar&) final;
 
+    void updateScrollerStyle() final;
+
+    void updateScrollerStyleForScrollableAreas(const WeakHashSet<ScrollableArea>*) final;
```

Comment:
Adding this here makes the responsibilities of `ScrollbarsController` less clear. Up until now, its role is to manage a single pair of scrollbars for a single ScrollableArea. I think you should preserve that, and not make it see ScrollableAreas that it does not manage.

Reply:
Right removed this.